src/com/lordsutch/android/signaldetector/SignalDetector.java

```diff
@@ -348,6 +348,8 @@ private Boolean validPhysicalCellID(int pci) {
         return (pci >= 0 && pci <= 503);
     }
 
+    private Boolean validEARFCN(int earfcn) { return earfcn != Integer.MAX_VALUE; }
```

Comment:
might use fundamental boolean type here for efficiency (rather than the Boolean object)